src/core/components/link/stories.tsx

```diff
@@ -16,6 +16,9 @@ const priorityLinks = [
 	<Link priority="secondary" href="#">
 		Secondary
 	</Link>,
+	<Link iconSide="left" icon={<SvgArrowRightStraight />} href="#">
+		With icon
+	</Link>,
```

Comment:
An icon story is already featured [further down this file](https://github.com/guardian/source/blob/master/src/core/components/link/stories.tsx#L99) 🙂